src/states/node.rs

```diff
@@ -888,35 +883,6 @@ impl Node {
             .init(self.full_id.clone(), &self.gen_pfx_info, &log_ident)
     }
 
-    // Completes a Node startup process and allows a node to behave as a `ManagedNode`.
-    // A given node is considered "established" when it exists in `chain.our_info().members()`
-    // first node: this method is skipped entirely as it behaves as a Node from startup.
-    fn node_established(&mut self, outbox: &mut EventBox) {
-        self.peer_mgr.set_established();
-        outbox.send_event(Event::Connected);
-
-        trace!(
-            "{} Node Established. Prefixes: {:?}",
-            self,
-            self.chain.prefixes()
-        );
-
-        self.update_peer_states(outbox);
-
-        // Allow other peers to bootstrap via us.
-        if let Err(err) = self.crust_service.set_accept_bootstrap(true) {
-            warn!("{} Unable to accept bootstrap connections. {:?}", self, err);
-        }
-        self.crust_service.set_service_discovery_listen(true);
-
-        let backlog = mem::replace(&mut self.routing_msg_backlog, vec![]);
-        backlog
```

Comment:
I guess this was made unecessary because we pass in msg_backlog to initialize  msg_queue.
Something to be careful about if keeping 2 queues for Establishing.